### DISCUSSION TITLE
Add abbreviated day legend to Calendar components

### DISCUSSION THREAD

**Discussion by Larkenx** [plouc/nivo#2751] [Ideas]
**Is your feature request related to a problem? Please describe.**
It's not immediately clear what days in a calendar are which days of the week - Monday, Tuesday, etc

**Describe the solution you'd like**
I'd like to be able to add a secondary y-axis on the left side of the calendar that shows shortened days to indicate a Monday, Tuesday, Wednesday, etc.

Here's an example edited with preview/paint

![image](https://user-images.githubusercontent.com/12945799/67789304-206c4080-fa4a-11e9-9415-ea99e56a7f59.png)

Looks like https://github.com/plouc/nivo/issues/19 may have been opened and closed with this feature in mind

**Comment by marnett-git** [comment]
I would also appreciate this feature being added. Looking into implementing this myself in the meantime.

**Comment by nleapai** [comment]
Bump

+1 to having this feature available on Calendar like it is on Time Range. I'm also not seeing the ability to inject layers on a Calendar like you can with Bar or Line. Does anyone know of a way to achieve this currently?

**Comment by AshleyBoyd11** [comment]
+1 
This would be awesome to have on the calendar component. Currently I need to choose between having a days of the week legend or having month separation and I'd love to have both!

**Comment by plouc** [comment]
This sounds like an interesting feature to have.
I personally have other priorities atm, but if someone wants to give it a try, that would be awesome.
I'm also converting this to a discussion, as this isn't strictly an issue.

**Reply by AshleyBoyd11** [reply]
I had a quick go and raised a PR https://github.com/plouc/nivo/pull/2753
